cli/cmd/install.go

```diff
@@ -435,6 +438,10 @@ func (options *installOptions) recordableFlagSet() *pflag.FlagSet {
 		&options.identityOptions.clockSkewAllowance, "identity-clock-skew-allowance", options.identityOptions.clockSkewAllowance,
 		"The amount of time to allow for clock skew within a Linkerd cluster",
 	)
+	flags.BoolVar(
+		&options.OmitWebhookSideEffects, "omit-webhook-side-effects", options.OmitWebhookSideEffects,
+		"Omit the sideEffects flag in the webhook manifests, to run on Kubernetes versions less than 1.12",
```

Comment:
I think we should emphasize to the users that if they are on k8s pre-1.12, they must provide this flag, during install and upgrade. How about, _Omit the sideEffects flag in the webhook manifests. This flag must be provided during install or upgrade for Kubernetes pre 1.12_